Astro 8/8 release notes (#2663)
astronomer/docs: astro/connect-aws.md

5. On the **Associate permissions** page, accept the default settings and then click **Next**.

> 6. On the **Grant access to principals** page, select **Allow sharing only within your organization**, and then enter your Astro AWS account ID for your organization in the **Enter an AWS account ID** field. To get the Astro AWS account ID, go to your Organization settings in the Cloud UI, click **General**, and then copy the value in the **ID** column for the Astro AWS account you want to share the Resolver rule with.

Reviewer: You don't "click General" since it is the default page within Org Settings.
```suggestion
6. On the **Grant access to principals** page, select **Allow sharing only within your organization**, and then enter your Astro AWS account ID for your organization in the **Enter an AWS account ID** field. To get the Astro AWS account ID, go to your Organization settings in the Cloud UI. From the **General** page you can then find the value in the **ID** column for the Astro AWS account you want to share the Resolver rule with.
```

5. On the **Associate permissions** page, accept the default settings and then click **Next**.

6. On the **Grant access to principals** page, select **Allow sharing only within your organization**, and then enter your Astro AWS account ID for your organization in the **Enter an AWS account ID** field. 

>     To get the Astro AWS account ID, click the name of your Workspace in the upper left corner of the Cloud UI, then click **Organization Settings**. From the **General** page, copy the **Organization ID** for the Astro AWS account you want to share the Resolver rule with.

Author: I think there's another PR out to fix this process anyway, this should be fixed by that
